pychromecast/controllers/yleareena.py

```diff
@@ -0,0 +1,88 @@
+"""
+Controller to interface with the Yle Areena app namespace.
+"""
+
+from ..config import APP_YLEAREENA
+from .media import MediaController, STREAM_TYPE_BUFFERED, TYPE_LOAD, MESSAGE_TYPE
+
+
+class YleAreenaController(MediaController):
+    """ Controller to interact with Yle Areena app namespace. """
+
+    # pylint: disable=useless-super-delegation
+    def __init__(self):
+        super(YleAreenaController, self).__init__()
+
+        self.app_id = APP_YLEAREENA
+
+    # pylint: disable=too-many-arguments
+    def play_areena_media(
+        self,
+        entry_id,
+        audio_language="",
+        text_language="off",
+        current_time=0,
+        autoplay=True,
+        stream_type=STREAM_TYPE_BUFFERED,
+    ):
+        """
+        Play media with the entry id "entry_id".
+        This value can be found by loading a page on Areena, e.g. https://areena.yle.fi/1-50097921
+        And finding the kaltura player which has an id of yle-kaltura-player3430579305188-29-0_whwjqpry
+        In this case the entry id is 0_whwjqpry
+        """
+
+        # pylint: disable=too-many-locals
+        def app_launched_callback():
```

Comment:
This is not necessary, `BaseController.send_message()` will automatically launch the necessary app and wait for it to be launched before sending the message:
https://github.com/home-assistant-libs/pychromecast/blob/7372f8d24209aad9b5f7f61f5988f28994a7ec60/pychromecast/controllers/__init__.py#L76-L81

Reply:
Huh, that's right. I guess that should also make the new check in `quick_play.quick_play` redundant.

I did get a strange error telling me something like "chromecast is not ready" previously... I'll need to check that again.